app/uk/gov/hmrc/tai/repositories/EmploymentRepository.scala

```diff
@@ -37,168 +35,125 @@ class EmploymentRepository @Inject()(
   rtiConnector: RtiConnector,
   cacheConnector: CacheConnector,
   npsConnector: NpsConnector,
-  auditor: Auditor) {
+  employmentBuilder: EmploymentBuilder) {
 
   private val EmploymentMongoKey = "EmploymentData"
 
   def employment(nino: Nino, id: Int)(
-    implicit hc: HeaderCarrier): Future[Either[EmploymentRetrievalError, Employment]] =
-    employmentsForYear(nino, TaxYear()) flatMap { empForYear =>
-      if (empForYear.exists(_.annualAccounts.exists(_.realTimeStatus == TemporarilyUnavailable))) {
-        Future.successful(Left(EmploymentAccountStubbed))
+    implicit hc: HeaderCarrier): Future[Either[EmploymentRetrievalError, Employment]] = {
+    val taxYear = TaxYear()
+    employmentsForYear(nino, taxYear) map { empForYear =>
+      if (empForYear.exists(_.tempUnavailableStubExistsForYear(taxYear))) {
```

Comment:
Same as containsTempAccount?

Reply:
Yes. I have pushed the Employments type higher as you mentioned in another comment.